packages/sources/finnhub-test/src/endpoint/router.ts

```diff
@@ -0,0 +1,56 @@
+import { RoutingTransport } from '@chainlink/external-adapter-framework/transports/meta'
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { httpTransport } from './quote'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { customSettings } from '../config'
+
+export const inputParameters = {
+  base: {
+    aliases: ['from', 'coin'],
+    type: 'string',
+    description:
+      'The symbol of the currency to query. The full list of options can be found here [Physical Currency list](https://www.alphavantage.co/physical_currency_list/) or [Cryptocurrency list](https://www.alphavantage.co/digital_currency_list/)',
+    required: true,
+  },
+} as const
```

Comment:
After the typescript upgrade and the fw bump past 0.25, we want to move to using the satisfies operator here like 
```suggestion
} satisfies InputParameters
```

Reply:
Done

Reply:
I think this is still missing